### PR TITLE
page rendering changes + moving TS code 

### PR DESCRIPTION
**Pull request by qiraahmad**

### CODE REVIEW

File: src/lib/fetchData.ts [hunk 1]
```diff
@@ -34,7 +35,14 @@ export async function fetchDeathsData() {
         const pageData = await fetchPage(page);
         allRows.push(...pageData);
     }
+    
+    const result = allRows.map((death){
+        death.modified_name = toKebabCase(death.name_en);
```
Comment (sinnytk) [inline src/lib/fetchData.ts]: Code breaks here. Missing arrow for function declaration 
```suggestion
    
    const result = allRows.map((death) => {
        death.modified_name = toKebabCase(death.name_en);
```

File: src/lib/fetchData.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { NOCODB_BASE } from "$env/static/private";
 import { nocodb_api } from "$lib/nocodb_api";
-import { getRandomSubset } from "./utils";
+import type { Person } from "../models/person";
+import { toKebabCase, getRandomSubset, capitaliseFirstLetter } from "./utils";
```
Comment (sinnytk) [inline src/lib/fetchData.ts]: code seems to break on this model import. am I missing some build command to generate the types?

**Review by sinnytk — COMMENTED** [review]
seems to break on build for me. let's see what's breaking before the review.

File: src/lib/fetchData.ts (hunk 2 above)
Comment (qiraahmad) [reply src/lib/fetchData.ts]: added the missing import, thanks for the catch

**Review by jokroese — APPROVED** [review]

**Pull request merged**